src/main/java/com/aws/greengrass/mqttbridge/BridgeConfig.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.mqttbridge;
+
+import com.aws.greengrass.componentmanager.KernelConfigResolver;
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.util.Coerce;
+import com.aws.greengrass.util.Utils;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Utilities for accessing MQTT Bridge configuration.
+ */
+public final class BridgeConfig {
+
+    @Deprecated
+    public static final String KEY_DEPRECATED_BROKER_URI = "brokerServerUri";
```

Comment:
keep constants as private as possible. I haven't read the rest of the change, but if these are only used in this file they should be private.

Reply:
a few of the constants are used in other classes.  the intent behind this class is to be a one-stop shop for these constants, so made them public to encourage their usage in the future.    I feel like inconsistent visibility would be a bit confusing, but happy to make the adjustments if you think otherwise

Reply:
update: reduced the visibility of most of the constants in latest commit